### ISSUE TITLE
Unable to compile: telemetry.hpp: No such file or directory

### ISSUE THREAD

**Issue by FestplattenSchnitzel** [CESNET/ipfixprobe#224]
Running

```sh
git clone --recurse-submodules https://github.com/CESNET/ipfixprobe
cd ipfixprobe
autoreconf -i
./configure
make
```

as described in the README results in failure during the `make` step:

```
make  all-recursive
make[1]: Entering directory '/opt/ipfixprobe/ipfixprobe'
Making all in .
make[2]: Entering directory '/opt/ipfixprobe/ipfixprobe'
  CXX      input/ipfixprobe-input.o
In file included from input/input.cpp:26:
./include/ipfixprobe/input.hpp:35:10: fatal error: telemetry.hpp: No such file or directory
   35 | #include <telemetry.hpp>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [Makefile:2105: input/ipfixprobe-input.o] Error 1
make[2]: Leaving directory '/opt/ipfixprobe/ipfixprobe'
make[1]: *** [Makefile:3064: all-recursive] Error 1
make[1]: Leaving directory '/opt/ipfixprobe/ipfixprobe'
make: *** [Makefile:1044: all] Error 2
```

**Comment by FestplattenSchnitzel** [comment]
https://github.com/CESNET/ipfixprobe/blob/c4019b7c64f004cfd8742341437102a1271c7205/.gitignore#L97

Everything named "ipfixprobe" gets ignored (also `include/ipfixprobe/`), so that's b/c the file was forgotten to push in the first place, I guess.

**Comment by SiskaPavel** [comment]
Solved in #242. 
Telemetry is our external library and can be installed from [COPR repository](https://copr.fedorainfracloud.org/coprs/g/CESNET/NEMEA-stable/package/telemetry/) or built from [source code](https://github.com/CESNET/telemetry)
